docs/services/data/capture-sync.md

```diff
@@ -49,47 +57,167 @@ However, in practice, high frequency data collection (> 100Hz) requires special
 {{% /tab %}}
 {{< /tabs >}}
 
-You can change the frequency of data capture at any time for individual resources.
-If you use {{< glossary_tooltip term_id="fragment" text="fragments" >}}, you can change the frequency of data capture in real time for some or all machines in a fleet at the resource or machine level.
+{{< expand "Click for an example." >}}
+Consider a tomato picking robot with a 3D camera and an arm.
+When you configure the robot, you might set the camera to capture point cloud data at a frequency of 30Hz.
+For the arm, you might capture joint positions at 1Hz.
 
-For example, consider a tomato picking robot with a 3D camera and an arm.
-When you configure the robot, you may set the camera to capture point cloud data at a frequency of 30Hz.
-For the arm, you may want to capture joint positions at 1Hz.
-If your requirements change and you want to capture data from both components at 10Hz, you can change the configurations at any time by changing the number.
+If your requirements change and you want to capture data from both components at 10Hz, you can change the capture rate at any time in each component's data capture configuration.
+{{< /expand >}}
 
-Data capture is frequently used with [Cloud Sync](/services/data/cloud-sync/).
-However, if you want to manage your machine's captured data yourself, you can enable only data capture without cloud sync.
+### Supported components and services
+
+The following components and services support data capture:
+
+{{< tabs >}}
+{{% tab name="viam-server" %}}
+
+| Type                           |
+| ------------------------------ |
+| Arm                            |
+| Board                          |
+| Camera                         |
+| Encoder                        |
+| Gantry                         |
+| Motor                          |
+| Movement sensor (includes GPS) |
+| Sensor                         |
+| Servo                          |
+| Vision service                 |
+
+{{% /tab %}}
+{{% tab name="viam-micro-server" %}}
+
+<!-- prettier-ignore -->
+| Type | Method |
+| ---- | ------ |
+| [Movement Sensor](/components/movement-sensor/) | [`AngularVelocity`](/components/movement-sensor/#getangularvelocity), [`LinearAcceleration`](/components/movement-sensor/#getlinearacceleration), [`LinearVelocity`](/components/movement-sensor/#getlinearvelocity) |
+| [Sensor](/components/sensor/) | [`GetReadings`](/components/sensor/#getreadings) |
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Cloud sync details
+
+### Frequency
```

Comment:
I think this should be explained in the parameter table where it's being configured. We may need two parameter tables for each part of the config.

Reply:
okay I'll remove it here

Reply:
I don't think we do--I think we just need viam-server versus viam-micro-server, which I'm doing in tabs